### PR TITLE
Feat/spell check on content

### PR DESCRIPTION
**Pull request by galadd**
Description:
This pull request addresses #11 Spell check on content added by user. It aims to check that Experience, Education and Skill do not contain any spelling errors.

Key Changes:
The spell_check function has been written to utils.py, and function implementation to function experience, education and skill.

Benefits:
This would help situations where a user has made an error during inputs. The function returns the error, and its correction

Testing:
The test covers the format of return, which is to return the misspelled word alongside the sentence or phrase it was in

### CODE REVIEW

**Comment by galadd** [comment]
#11

**Review by Henoven — CHANGES_REQUESTED** [review]
It seems your solution should work and solve #11 However, I see that there are some things that you should address first, before approving this PR.

File: test_pytest.py [hunk 1]
```diff
@@ -1,125 +1,152 @@
...
-    # check if GET request with index gives the same JSON
-    response = app.test_client().get('/resume/experience')
-    assert response.json[ind1] == exp1
-    assert response.json[ind2] == exp2
-    assert response.json[ind3] == exp3
-
-
-def test_experience():
+from app import Experience, spell_check
+# from models import Experience
```
[the remark this answers is not in the document]
Comment (galadd) [reply test_pytest.py:5]: When I initially cloned the repo, and ran the test, it failed. So I commented it all out and focused only on the test for the function I'll be writing

File: utils.py [hunk 2]
```diff
@@ -48,3 +49,28 @@ def get_skill_by_index(data, index):
...
+
+
+def spell_check(data):
+    '''
+    Spell check on  content added by user
+    '''
+    spell = SpellChecker()
+    corrections = []
+
+    for attr in data:
```
[the remark this answers is not in the document]
Comment (galadd) [reply utils.py:61]: > @galadd What is `attr` standing for?

attr represents the whole field for Experience, Education or Skill

**Pull request closed without merging**